commands/update_dependencies.go

```diff
@@ -53,8 +55,6 @@ func updateDependenciesRun(flags updateDependenciesFlags) error {
 		originalVersions[d.Version] = d.Version
 	}
 
-	api := flags.api
-
 	// if a metadata file is provided, use that as the version and metadata
 	// source
 	if flags.metadataFile != "" {
```

Comment:
@sophiewigmore if `metadata-file` is required, can we drop the `if` condition too?